src/core/agent/services/signifyNotificationService.ts

```diff
@@ -158,7 +158,38 @@ class SignifyNotificationService extends AgentService {
     notif: Notification,
     callback: (event: KeriaNotification) => void
   ) {
-    // We only process with the credential and the multisig at the moment
+    if (notif.a.r === NotificationRoute.MultiSigRpy) {
```

Comment:
A `Rpy` message can be used for other things. Is there anything in the message that indicates it's an end role authorsation?

Reply:
I will use this path to check it `exn.e.rpy.r`

Reply:
What's the value of that?

Reply:
"/end/role/add"

Reply:
OK - so we check that `exn.e.rpy.r` is that value so. So here:
- we get the exchange message
- check for `/end/role/add`
- call `joinAuthorization`
- return

I think we can change the signature of `joinAuthorization` to be `(notificationSaid: string, multisigAid: string)` - since we fetch it here to check `/end/role/add` no need to fetch it again in `joinAuthorization`.

Reply:
Actually, it is not the aid, I am getting it from groups().getRequest() full path is `multisigNotification[0].exn.e.rpy.r`. Here is it:
`const multisigNotification = await this.props.signifyClient
        .groups()
        .getRequest(notif.a.d)`

Reply:
OK - then we can get that here, and pass `exn` to `joinAuthorisation` right?
```
const exn = request[0].exn;
```

Do we need to call `exchanges().get`? Does `groups() .getRequest` have all of the info we need?

Reply:
I just checked, `groups() .getRequest` has all the info we need